docs/admin/federation/index.md

```diff
@@ -15,72 +16,129 @@ This guide explains how to set up cluster federation that lets us control multip
 This guide assumes that we have a running Kubernetes cluster.
 If not, then head over to the [getting started guides](/docs/getting-started-guides/) to bring up a cluster.
 
-This guide also assumes that we have the Kubernetes source code that can be
-[downloaded from here](/docs/getting-started-guides/binary_release/).
+This guide also assumes that we have a Kubernetes release
+[downloaded from here](/docs/getting-started-guides/binary_release/),
+extracted into a directory and all the commands in this guide are run from
+that directory.
+
+```shell
+$ curl -L https://github.com/kubernetes/kubernetes/releases/download/v1.4.0/kubernetes.tar.gz | tar xvzf -
```

Comment:
`{{page.version}}` should give you `v1.4`.
Found from: https://github.com/kubernetes/kubernetes.github.io/blob/913981e3236b76e409af95fcba0a323657bddea0/docs/samples.md